src/DacpacTool/Extensions.cs

```diff
@@ -16,6 +16,39 @@ namespace MSBuild.Sdk.SqlProj.DacpacTool
     public static class Extensions
 #pragma warning restore CA1724 // Type names should not match namespaces
     {
+        private static Type GetCustomSchemaDataType()
```

Comment:
I was thinking more about a static constructor that resolves these types and methods and puts them into static fields that can then be used

Reply:
Hmm, interesting, not allowed by what?